aspnetcore/performance/response-compression.md

```diff
@@ -36,16 +36,33 @@ Usually, any response not natively compressed can benefit from response compress
 
 When a client can process compressed content, the client must inform the server of its capabilities by sending the `Accept-Encoding` header with the request. When a server sends compressed content, it must include information in the `Content-Encoding` header on how the compressed response is encoded. Content encoding designations supported by the middleware are shown in the following table.
 
-| `Accept-Encoding` header values | Middleware Supported | Description                                                 |
-| ------------------------------- | :------------------: | ----------------------------------------------------------- |
-| `br`                            | No                   | Brotli Compressed Data Format                               |
-| `compress`                      | No                   | UNIX "compress" data format                                 |
-| `deflate`                       | No                   | "deflate" compressed data inside the "zlib" data format     |
-| `exi`                           | No                   | W3C Efficient XML Interchange                               |
-| `gzip`                          | Yes (default)        | gzip file format                                            |
+::: moniker range=">= aspnetcore-2.2"
+
+| `Accept-Encoding` header values | Middleware Supported | Description |
+| ------------------------------- | :------------------: | ----------- |
+| `br`                            | Yes (default)        | [Brotli compressed data format](https://tools.ietf.org/html/rfc7932) |
+| `deflate`                       | No                   | [DEFLATE compressed data format](https://www.ietf.org/rfc/rfc1951.txt) |
```

Comment:
Maybe all of these should use the `tools.ietf.org` links? Or at least be consistent?

Reply:
Thanks ... yes, I'll do that. I was relying too heavily on search results, and top results were inconsistent.

Reply:
@khellang I found one I could change to `www.ietf.org` but not the other two. I'll see if John has any further feedback about it on review. Thanks again for taking a look.